DeepPanel/DeepPanel.swift

```diff
@@ -82,7 +82,7 @@ public class DeepPanel {
               options?.threadCount = 2
         #else
               // Use GPU on real device for inference as this model is fully supported.
-              delegates = [MetalDelegate()]
+              delegates = useMetal ? [MetalDelegate()] : nil
```

Comment:
Wouldn't you want to
> // Use CPU for inference as MetalDelegate does not support iOS simulator.

in case Metal is not supported? As in, I would expect that you need to use something for inference and this option is using `nil` if Metal isn't supported on a real device.

Reply:
As far as I know, Metal is a replacement for OpenGL and does not require any particular hardware. Any device running iOS 8 and higher can run Metal so, unless I'm mistaken, we won't find any device not supporting Metal. [Reference](https://developer.apple.com/library/archive/documentation/DeviceInformation/Reference/iOSDeviceCompatibility/HardwareGPUInformation/HardwareGPUInformation.html#//apple_ref/doc/uid/TP40013599-CH106-SW1)

This flag is just a workaround until we understand better why sometimes we don't get any results when using Metal. 

It's true that, when the flag is `true`, the comment above is not accurate. But I think we'll be able to completely get rid of the flag once we know the details.